packages/bitcore-node/src/config.ts

```diff
@@ -79,7 +79,7 @@ const Config = function(): ConfigType {
       },
       p2p: {},
       socket: {
-        bwsKeys: []
+        bwsKeys: ['02064c30a69f2954fde1a32355c3769cd812ae4339d814835467d0aed398617244']
```

Comment:
Adding a default key so bws and bitcore-node can talk to each other on clean installs